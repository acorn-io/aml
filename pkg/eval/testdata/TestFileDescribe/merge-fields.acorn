args: {
    // First
    key: "value1"
}

args: {
    // Second
    key: "value2"
}
