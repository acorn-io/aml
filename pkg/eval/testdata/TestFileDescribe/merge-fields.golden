{
  "args": {
    "path": "args",
    "fields": [
      {
        "name": "key",
        "description": "First\nSecond",
        "type": {
          "kind": "string",
          "constraints": [
            {
              "op": "type",
              "right": {
                "kind": "string",
                "default": "value1"
              }
            },
            {
              "op": "type",
              "right": {
                "kind": "string",
                "default": "value2"
              }
            }
          ],
          "default": "value1"
        }
      }
    ],
    "allowNewKeys": true
  }
}